universal-application-tool-0.0.1/app/views/admin/questions/QuestionsListView.java

```diff
@@ -111,22 +106,24 @@ private Tag renderAddQuestionLink() {
     return linkButton.with(dropdown);
   }
 
-  private Tag renderSummary(ImmutableList<QuestionDefinition> questions) {
-    return div("Total Questions: " + questions.size())
+  private Tag renderSummary(ActiveAndDraftQuestions activeAndDraftQuestions) {
+    return div("Total Questions: " + activeAndDraftQuestions.getActiveSize())
```

Comment:
Thought (non-blocking): should this also include draft questions?

Reply:
Sure, added!